apps/accounts/views.py

```diff
@@ -1,20 +1,34 @@
 from django.contrib.auth import logout
 
+from allauth.account.utils import send_email_confirmation
+
 from rest_framework.response import Response
 from rest_framework import permissions, status
 from rest_framework.decorators import (api_view,
                                        authentication_classes,
-                                       permission_classes,)
+                                       permission_classes,
+                                       throttle_classes,)
 from rest_framework_expiring_authtoken.authentication import (ExpiringTokenAuthentication,)
 
+from .throttles import ResendEmailThrottle
+
 
 @api_view(['POST'])
 @permission_classes((permissions.IsAuthenticated,))
 @authentication_classes((ExpiringTokenAuthentication,))
 def disable_user(request):
-
     user = request.user
     user.is_active = False
     user.save()
     logout(request)
     return Response(status=status.HTTP_200_OK)
+
+
+@throttle_classes([ResendEmailThrottle])
+@api_view(['POST'])
+@permission_classes((permissions.IsAuthenticated,))
+@authentication_classes((ExpiringTokenAuthentication,))
+def resend_email(request):
```

Comment:
Can you please add docstring to this API? Also, please rename this to `resend_email_confirmation`